### DISCUSSION TITLE
Client Side can't disconnect or dispose

### DISCUSSION THREAD

**Discussion by boxu2018** [jchristn/SuperSimpleTcp#222] [General]
once client has been connected with server, i'd like to disconnect the client,
so use method :client.disconnect() or client.dispose();
there will throw exception, like this: System.OperationCanceledException:“The operation was canceled.”

**Comment by jchristn** [comment]
Can you supply a stack trace?  I've added some catches for both OperationCanceledException and TaskCanceledException going into 3.0.17.
